### DISCUSSION TITLE
Dependency Dashboard list multi packages should be updated but only create tow MRs

### DISCUSSION THREAD

**Discussion by radicalviva** [renovatebot/renovate#20599] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.0.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

gitlab

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

<img width="761" alt="image" src="https://user-images.githubusercontent.com/15885774/220984202-5f51707c-8bff-4ad5-9d01-2a56073b5555.png">
issue list multi packages, but only create part MRs
Here is my renovate.json config
`
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["config:base"]
}
`

### What you tried so far.

When I trigger the renovate again, it also created two new MRs, so it has some limit rules ?
If I miss some config ?

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Your screenshot even shows 4 MRs, not 2. Most likely you are seeing 2 MRs created per hour which is the rate limit from `config:base` (and has been for years). If you were to look at the logs, you'd probably see that confirmed.
